.gitconfig

```diff
@@ -57,6 +57,13 @@
 	# a.k.a. ‘delete merged’
 	dm = "!git branch --merged | grep -v '\\*' | xargs -n 1 git branch -d"
 
+	# Instantly clean a git repo from extraneous files in a non-blocking fashion
+	purge = "!f() { \
+		declare tmp_dir=$(mktemp -d); \
+		git clean -fdxn | sed -n -r 's/^Would remove (.*)/\\1/p' | xargs mv -t \"$tmp_dir\"; \
```

Comment:
Pretty sure you do need to. I would leave it quoted.

If you want a real answer, put the entire function in a shell script file and run [shellcheck](/koalaman/shellcheck) against it.

For compatibility with non-Bash (`sh` and up), you can check for *Bashisms* using [checkbashisms](https://anonscm.debian.org/cgit/collab-maint/devscripts.git/tree/scripts/checkbashisms.pl).